site/src/pages/[shortname]/[blockSlug].page.tsx

```diff
@@ -277,14 +297,54 @@ const BlockPage: NextPage<BlockPageProps> = ({
           </Box>
         </Box>
 
-        <Box sx={{ mb: 15 }}>
+        <Box sx={{ mb: 10 }}>
           <BlockDataContainer
             metadata={blockMetadata}
             schema={schema}
             blockModule={blockModule}
           />
         </Box>
 
+        {repositoryUrl && (
+          <Box
+            mb={10}
+            sx={{
+              display: "grid",
+              gridTemplateColumns: { xs: "1fr", md: "60% 40%" },
+            }}
+          >
+            <Box />
+            <Box sx={{ overflow: "hidden" }} pl={{ xs: 0, md: 2 }}>
+              <Typography
+                variant="bpLargeText"
+                sx={{
+                  fontWeight: "bold",
+                  color: theme.palette.gray["70"],
+                  marginBottom: 2,
+                }}
+              >
+                Repository
+              </Typography>
+              <Box sx={{ display: "flex" }}>
+                <Box
+                  component="img"
+                  alt="GitHub Link"
+                  sx={{ marginRight: 1.5 }}
+                  src="/assets/link.svg"
+                />{" "}
+                <Typography
+                  variant="bpSmallCopy"
+                  sx={{ overflow: "hidden", textOverflow: "ellipsis" }}
+                >
+                  <Link href={repositoryUrl.href}>
+                    {repositoryUrl.hostname}/{repositoryUrl.pathname}
```

Comment:
This is introducing an extra slash
<img width="153" alt="Screenshot 2022-02-07 at 10 46 20" src="https://user-images.githubusercontent.com/37743469/152773777-83221102-bf3d-432f-82be-88df6bde6117.png">